app/models/setting/remote_execution.rb

```diff
@@ -63,7 +63,11 @@ def self.load_defaults
                  nil,
                  N_('Default SSH key passphrase'),
                  nil,
-                 { :encrypted => true })
+                 { :encrypted => true }),
+        self.set('remote_execution_workers_pool_size',
```

Comment:
It should be probably state somwhere that the user has to restart httpd+foreman-tasks services in production deployment in order for the changes to come into effect.

Reply:
Good call, added the info into the description